changelog.d/gh-9628.fixed

```diff
@@ -0,0 +1,3 @@
+When Semgrep fails to produce a bytepos from a line/col position (an internal
```

Comment:
> When Semgrep fails to produce a bytepos from a line/col

Probably too much detail for the changelog, I think those details are for devs and belong to the PR description.